frontend/src/App.svelte

```diff
@@ -3,10 +3,14 @@
 	import { routes } from "../.routify/routes"; // this is a build time import
 
 	// graphql stuff
-	import { initClient, query, operationStore } from "@urql/svelte"
+	import { initClient, query, operationStore } from "@urql/svelte";
+
+	const prodGraphqlURL = `https://${window.location.host}/graphql`;
+	const devGraphqlURL = `http://localhost:3000/graphql`;
```

Comment:
Real talk. Do we really need this? On dev it would use localhost:port which is what we want. I don't think we would need it because the graphql url would just work. Not to mention having it hardcoded to 3000 makes the port in config not matter.

Reply:
Port was already hardcoded before this PR. But yeah ill fix